src/main/java/org/entando/plugins/pda/core/model/form/FormField.java

```diff
@@ -0,0 +1,20 @@
+package org.entando.plugins.pda.core.model.form;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@Data
+@AllArgsConstructor
+public class FormField {
+
+    private String id;
+    private String name;
+    private String label;
+    private Boolean required;
+    private Boolean readOnly;
```

Comment:
Change to `boolean`  so that lombok generates `isRequired()` and `isReadOnly()` methods.